WordPress/src/main/res/values/strings.xml

```diff
@@ -1455,6 +1455,15 @@
     <!-- Remote Post has conflicts with local post -->
     <string name="local_post_is_conflicted">Version conflict</string>
 
+    <!-- Post Remote Preview -->
+    <string name="post_preview_saving_draft">Saving&#8230;</string>
+    <string name="post_preview_remote_auto_saving_post">Generating Preview&#8230;</string>
+    <string name="remote_preview_operation_error">Error during remote preview preparation.\nPlease try again.</string>
```

Comment:
I think we can do better on the error wording, do you mind updating it to: `Error while trying to save post before preview` or something similar. I checked on iOS, they show the preview empty screen when an error happens. TBH I think it's better to show a Snackbar in the case of an error.

Reply:
I was also not happy with the wording actually 😅 ; your suggestion sounds good to me. Changed here 5951e1a694a041d01e1e20ba726b2e97c910bcbc. Also agree that snackbar logic can be better in case of a generic error like this.